src/main/java/org/stocker/routes/UpdateRecordsRoute.java

```diff
@@ -69,22 +83,39 @@ public void handle(YokeRequest yokeRequest, Handler<Object> handler) {
         response.end();
     }
 
-    public void updateStocksForDate(Date dateObj) throws StockDataNotFoundForGivenDateException, NseDataObjParseException {
-        ArrayList<String> processedStocksList = new ArrayList<>();
+    public void updateStocksForDate(StockMarket market, Date dateObj) throws StockDataNotFoundForGivenDateException, NseDataObjParseException, IOException {
+        // TODO: Move these operations to some country specific adapters
+        if (market == StockMarket.INDIA) {
+            ArrayList<String> processedStocksList = new ArrayList<>();
+            List<NseDataObj> dataObjList;
+            dataObjList = stockDataClient.getStockData(dateObj, ReportType.BHAV_REPORT);
+            List<NseDataObj> validStocks = dataObjList.stream()
+                    .filter(dataObj -> StockValidityCheckerService.isDataValid(dataObj)
+                            && isValidAsPerInstrument(dataObj))
+                    .collect(Collectors.toList());
 
-        List<NseDataObj> dataObjList;
-        dataObjList = stockDataClient.getStockData(dateObj, ReportType.BHAV_REPORT);
-        List<NseDataObj> validStocks = dataObjList.stream()
-                                        .filter(dataObj -> StockValidityCheckerService.isDataValid(dataObj)
-                                                && isValidAsPerInstrument(dataObj))
-                                        .collect(Collectors.toList());
+            for(NseDataObj obj : validStocks){
+                if(processedStocksList.contains(obj.rowData.get(SYMBOL))){
+                    continue;
+                }
+                stockDBClient.insert(obj);
+                processedStocksList.add(obj.rowData.get(SYMBOL));
+            }
+        } else {
+            //TODO : Find an API which will give all US stocks and use that
+            List<String> usStockSymbols = Arrays.asList("TSLA", "GOOG");
+            Calendar from = Calendar.getInstance();
+            from.setTime(dateObj);
 
-        for(NseDataObj obj : validStocks){
-            if(processedStocksList.contains(obj.rowData.get(SYMBOL))){
-                continue;
+            ArrayList<NasdaqDataObj> stocks = new ArrayList<>();
+            for (String stockSymbol : usStockSymbols) {
+                Stock stock = YahooFinance.get(stockSymbol, from, Interval.DAILY);
+                NasdaqDataObj nasdaqDataObj = new NasdaqDataObj(stock);
+                System.out.println(nasdaqDataObj.toString());
+                stocks.add(nasdaqDataObj);
             }
-            stockDBClient.insert(obj);
-            processedStocksList.add(obj.rowData.get(SYMBOL));
+
+            //TODO: Save the stocks in DB
```

Comment:
Yes surely need to think of this, its better to think of this now otherwise we might require additional work in the future to migrate these SYMBOL's to support multiple markets. I was going to handle this once I got to this TODO. This PR I am just getting the data and not storing it.